textual/src/toga_textual/widgets/base.py

```diff
@@ -0,0 +1,170 @@
+from travertino.size import at_least
+
+from toga.style.pack import ROW
+
+
+# We assume a terminal is 800x600 pixels, mapping to 80x25 characters.
+# This results in an uneven scale in the horizontal and vertical directions
+class Scalable:
+    HORIZONTAL_SCALE = 10
+    VERTICAL_SCALE = 25
```

Comment:
600 // 25 == 24. Suggest making the calculation explicit:

```suggestion
    HORIZONTAL_SCALE = 800 // 80
    VERTICAL_SCALE = 600 // 25
```

Reply:
Good catch on the 24 lines.

Reply:
What I meant was that 600 / 25 gives 24 pixels per line, not 25. I don't think subtracting one line for the title bar really makes sense, because the title bar is included in the hypothetical default terminal height of 600 pixels.

Reply:
You're right - I was overthinking things. The aspect ratio mapping of of 80x25->800x600 shouldn't be excluding the titlebar.